### PR TITLE
Update flow 72

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this updates flow to 72. They are moving to deprecate *.  My approach was generally
1. use the correct type (a ton of promises we just typed as * but really they had types)
1. use any (often we actually don't care about that types, often a component is passed and we don't care about its props / state)
1. flowissue to keep the star, we can debate this
1. use any. in these cases flow is likely actually falling back to this anyways

i think having flow coverage in our prs would help making better decisions here

**Comment by chrisnojima** [comment]
oh i updated the code gen and removed More.returnType and used $Call instead

File: shared/actions/json/profile.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "prelude": [
-    "import * as Types from '../constants/types/profile'"
+    "import * as Types from '../constants/types/profile'",
```
Comment (akalin-keybase) [inline shared/actions/json/profile.json]: still needed?

File: shared/actions/json/chat2.json [hunk 2]
```diff
@@ -262,8 +262,8 @@
     "sendToPendingConversation": {
       "users": "Array<string>",
       "sendingAction": [
-        "More.ReturnType<typeof createMessageSend>",
-        "More.ReturnType<typeof createAttachmentUpload>"
+        "MessageSendPayload",
```
Comment (akalin-keybase) [inline shared/actions/json/chat2.json]: importing More still needed in this file?

File: shared/actions/json/profile.json (hunk 1 above)
Comment (chrisnojima) [reply shared/actions/json/profile.json]: ? yes it uses More. those types should move out of there but keeping it small for now

File: shared/actions/json/chat2.json (hunk 2 above)
Comment (chrisnojima) [reply shared/actions/json/chat2.json]: ? it doesn't refer to more

**Pull request merged**